.pre-commit-config.yaml

```diff
@@ -48,18 +41,9 @@ repos:
         files: src
         args: []
         additional_dependencies:
-          - numpy~=1.24.0
+          - numpy
           - packaging
 
-  - repo: https://github.com/PyCQA/flake8
-    rev: 6.0.0
-    hooks:
-      - id: flake8
-        additional_dependencies:
-          - flake8-bugbear
-          - flake8-docstrings
```

Comment:
For some reason, Ruff's "D" code catches a lot of stuff the old one didn't. So I've removed it for now, and we can enable new codes (including "D") in the future.